src/System.Net.Http.WinHttpHandler/tests/project.lock.json

```diff
@@ -173,10 +173,10 @@
           "lib/dotnet/System.IO.FileSystem.Primitives.dll": {}
         }
       },
-      "System.Linq/4.0.0-beta-22816": {
+      "System.Linq/4.0.0-beta-22703": {
         "dependencies": {
-          "System.Collections": "4.0.10-beta-22816",
-          "System.Runtime": "4.0.20-beta-22816"
+          "System.Collections": "4.0.0-beta-22703",
+          "System.Runtime": "4.0.0-beta-22703"
```

Comment:
I updated them all to beta, and one of them is still rolling back. But as far as I can tell, that's "right", because that Linq dependency is coming from xunit, and that appears to be the version that xunit.netcore.extensions references... so that's correct?